### PR TITLE
添加已有的播客节目

### PR DESCRIPTION
**Pull request by kang8**
Close: #24, #39, #35

### CODE REVIEW

**Review by LinuxSuRen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -17,6 +17,9 @@
 | S01E03 | [专访 YiHong，自学成为流行开源项目作者的点滴](https://www.ximalaya.com/sound/474215643) | [YiHong](https://github.com/yihong0618) | [Wey Gu](https://github.com/wey-gu) |
 | S01E04 | [Miya 的故事｜从工地打印小妹，到全球知名教育公益开源社区大使](https://www.ximalaya.com/sound/484328758) | [Miya](https://github.com/miyaliu666) | [Rick](https://github.com/linuxsuren)、[majinghe](https://github.com/majinghe) |
 | S01E05 | [专访 琚[jū]致远：聊聊兴趣与成就感如何驱动他参与开源社区](https://www.ximalaya.com/sound/493354361) | [琚[jū]致远](https://github.com/juzhiyuan) | [Wey Gu](https://github.com/wey-gu)、[majinghe](https://github.com/majinghe)、[Rick](https://github.com/linuxsuren) |
+| S01E06 | [从民营书店进货员到开源创业工程师的故事](https://www.ximalaya.com/sound/510054673) | [水歌](https://github.com/TechQuery) | [Rick](https://github.com/linuxsuren) |
+| S01E07 | [面对开放的大门，技术小白可以何种姿势参与开源](https://www.ximalaya.com/sound/511091248) | [李梦](https://github.com/LiMeng00) | [Rick](https://github.com/linuxsuren) |
```
Comment (juzhiyuan) [inline README.md:21]: Just to make sure, is `可以何种` correct?

Comment (LinuxSuRen) [reply README.md:21]: 读着是有点奇怪

Comment (wey-gu) [reply README.md:21]: How about this?
```diff
- 技术小白可以何种姿势参与开源
+ 技术小白可以如何开始参与开源
```

**Review by LinuxSuRen — APPROVED** [review]

**Review by wey-gu — APPROVED** [review]
Great Job, Thanks!

**Comment by LinuxSuRen** [comment]
image

**Pull request merged**